src/OrchardCore/Orchard.DisplayManagement/Fluid/Statements/DisplayStatement.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Fluid;
+using Fluid.Ast;
+
+namespace Orchard.DisplayManagement.Fluid.Statements
+{
+    public class DisplayStatement : Statement
+    {
+        public DisplayStatement(Expression shape)
+        {
+            Shape = shape;
+        }
+
+        public Expression Shape { get; }
+
+        public override async Task<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context)
+        {
+            var page = FluidViewTemplate.EnsureFluidPage(context, "Display");
+            var shape = (await Shape.EvaluateAsync(context)).ToObjectValue();
+
+            await writer.WriteAsync((await page.DisplayAsync(shape)).ToString());
```

Comment:
Might be better like this to prevent allocations:

```
            var htmlContent = await page.DisplayAsync(shape);
            htmlContent.WriteTo(writer, HtmlEncoder.Default);
```